app/javascript/components/recurrence-form.js

```diff
@@ -26,7 +26,7 @@ class RecurrenceForm {
       this.intervalTarget.value = model.interval;
       if (model.until) {
         // Ignore the timezone in the “until” date. See #1339
-        this.untilTarget.value = moment(model.until, 'YYYY-MM-DD').format("DD/MM/YYYY");
+        this.untilTarget.value = moment(model.until).format('YYYY-MM-DD')
```

Comment:
Il me semble que la CI passe si on arrête d'ignorer la timezone ici
```suggestion
        this.untilTarget.value = moment(model.until).tz("Europe/Paris").format('YYYY-MM-DD')
```

Reply:
Disons que là, ça CI passe de toute façon, avec ça ou sans ça. Ce que tu dis, c'est que nous pourrions remettre en place la vérification des dates du coup ?